CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-0.22.0 / 2017-12-12
+0.23.0 / 2017-12-14
```

Comment:
why are we skipping `0.22.0` ?

Reply:
It has already been created.

Reply:
aah `0.22.0` was `Pre-release` I thought it was just `Draft`. All good then.